File: Objetivo 3/Objetivo3_Exc_L2_Java/src/model/Bicicleta.java
package model;

public class Bicicleta extends Veiculo{
	private int tamanhoRoda;
	private int chassi;
	
	public int getTamanhoRoda() {
		return tamanhoRoda;
	}
	public void setTamanhoRoda(int tamanhoRoda) {
		this.tamanhoRoda = tamanhoRoda;
	}
	public int getChassi() {
		return chassi;
	}
	public void setChassi(int chassi) {
		this.chassi = chassi;
	}
	@Override
	public String toString() {
		return "Bicicleta [ Modelo= " + getModelo() + ", Marca=" + getMarca()+ ", tamanhoRoda=" + tamanhoRoda + ", chassi=" + chassi + ", NumeroDeEixos="
				+ getNumeroDeEixos() + ", Propulsao=" + getPropulsao()	+ ", AnoFabricacao=" + getAnoFabricacao() + "]";
	}
	
	
	
	
}

File: Objetivo 3/Objetivo3_Exc_L2_PHP/model/Bicicleta.php
<?php

/**
 * Description of Bicleta
 *
 * @author vpsil
 */
class Bicleta extends Veiculo{
    private $tamanhoDaRoda;
    private $chassi;
    
    function __construct($numDeEixos, $propulsao, $marca, $modelo, $anoFabricacao, $tamanhoDaRoda, $chassi) {
        parent::__construct($numDeEixos, $propulsao, $marca, $modelo, $anoFabricacao);
        $this->tamanhoDaRoda = $tamanhoDaRoda;
        $this->chassi = $chassi;
    }
    
    function getTamanhoDaRoda() {
        return $this->tamanhoDaRoda;
    }

    function getChassi() {
        return $this->chassi;
    }

    function setTamanhoDaRoda($tamanhoDaRoda) {
        $this->tamanhoDaRoda = $tamanhoDaRoda;
    }

    function setChassi($chassi) {
        $this->chassi = $chassi;
    }

    public function __toString() {
        return "Bicicleta[Tamanho da Roda=$this->tamanhoDaRoda, Chassi=$this->chassi " . parent::__toString();
    }


}
